app/script/conversation/ConversationRepository.js

```diff
@@ -223,6 +223,14 @@ z.conversation.ConversationRepository = class ConversationRepository {
     );
     amplify.subscribe(z.event.WebApp.TEAM.MEMBER_LEAVE, this.teamMemberLeave.bind(this));
     amplify.subscribe(z.event.WebApp.USER.UNBLOCKED, this.unblocked_user.bind(this));
+
+    this.eventService.addEventUpdatedListener(this._updateLocalMessageEntity.bind(this));
```

Comment:
This is a first attempt to have the view listening to DB event to update what is displayed. 

In the future, we can migrate all the DB updates (namely: link previews, edits, upload(fail/success)) to that single entry point